lib/css/atomic/_stacks-misc.less

```
@@ -80,6 +80,18 @@
 }
 
+//  ============================================================================
+//  $  OBJECT-FIT / OBJECT-POSITION
+//  ----------------------------------------------------------------------------
+
+.of-contain { object-fit: contain !important; }
+.of-cover { object-fit: cover !important; }
+.of-fill { object-fit: fill !important; }
+.of-none { object-fit: none !important; }
+.of-scale-down { object-fit: scale-down !important; }
+
+.op-center { object-position: 50% 50% !important; }
```

```suggestion
.op-center { object-position: center !important; }
```

This suggestion is just a _very_ semantic one, but I tend to prefer the value to match the class name when possible.

---

I have a vague memory of `object-position: center` not actually working last time I tried it

---

`center` should work just the same. Tested locally and seemed fine.

Which makes me realize we should also add an example for `.op-center`. I'll pop one in here.